src/Lykke.Service.OAuth/Extensions/OpenIdConnectRequestExtensions.cs

```diff
@@ -11,5 +13,20 @@ public static string GetAcrValue(this OpenIdConnectRequest request, string key)
             var value = acrValues.FirstOrDefault(s => s.StartsWith($"{key}:"));
             return value?.Substring(key.Length + 1);
         }
+
+        public static string Serialize(this OpenIdConnectRequest request)
+        {
+            var serialized = request.ToJson();
```

Comment:
Better to use  to avoid null reference
```suggestion
            var serialized = JsonConvert.SerializeObject(request);
```

Reply:
https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/JsonSerialisersExt.cs#L28
this code is under the hood